addons/project/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_project_task_company_id(cr):
+    openupgrade.logged_query(
+        cr, """
```

Comment:
Hi,

Are we sure on this? To me this should be the last resort if all the other related fields with company ids are NULL.

company_id of project should be first choice, then assigned user, then assigned manager.

But also, I don't know the scope for adding notes/docs but telling people to set company correctly first for tasks pre-migration.

Reply:
You are right `project.project` is default `company_id` and I changed it.
But I have no idea about it should be in `pre-migration` or `post-migration`.
I thing fill should be in `pre-migration`

Reply:
it's ok as it is now

Reply:
It has to in pre-migration because in post-migration all tasks already have company_id.
And I don`t see the query for tasks without project.
<pre>

    openupgrade.logged_query(
        cr, """
        UPDATE project_task pt
        SET company_id = ru.company_id
        FROM res_users ru
        WHERE pt.project_id IS NULL AND pt.company_id IS NULL AND pt.create_uid = ru.id
        """
    )
</pre>